src/__e2e__/changelog.test.ts

```diff
@@ -189,5 +189,48 @@ describe('changelog generation', () => {
       const groupedChangelogText = fs.readFileSync(groupedChangelogFile, { encoding: 'utf-8' });
       expect(cleanMarkdownForSnapshot(groupedChangelogText)).toMatchSnapshot();
     });
+
+    it('Verify that the changeFile transform functions are run, if provided', async () => {
+      const monoRepo = monoRepoFactory.cloneRepository();
+      monoRepo.commitChange('foo');
+      writeChangeFiles({ foo: getChange() }, monoRepo.rootPath);
+
+      monoRepo.commitChange('bar');
+      writeChangeFiles({ bar: getChange({ packageName: 'bar', comment: 'comment 2' }) }, monoRepo.rootPath);
+
+      const beachballOptions = {
+        path: monoRepo.rootPath,
+        transform: {
+          changeFiles: (changeFile, path) => {
+            changeFile.comment = 'Edited comment for testing';
+            return changeFile;
+          }
+        },
+        changelog: {
+          groups: [
+            {
+              masterPackageName: 'foo',
+              changelogPath: path.join(monoRepo.rootPath, 'packages', 'foo'),
+              include: ['packages/foo', 'packages/bar'],
+            },
+          ],
+        },
+      } as BeachballOptions;
```

Comment:
Is this to work around typing issues? Curious if `const beachballOptions: BeachballOptions` didn't work

Reply:
`const beachballOptions: BeachballOptions` does a more stricter type check and we then had to provide all the mandatory fields. So, chose doing this way.

Reply:
In that case you should do `const beachballOptions: Partial<BeachballOptions>` as it is still more typesafe than casting. It will flag any incorrectly typed or extraneous property names.